src/MongoDB.Bson/ObjectModel/BsonBinaryData.cs

```diff
@@ -149,9 +170,17 @@ public byte[] Bytes
         /// <summary>
         /// Gets the representation to use when representing the Guid as BSON binary data.
         /// </summary>
+        [Obsolete("This property will be removed in a later release.")]
```

Comment:
AFAIK the C# driver is the only one that had this concept of "tagging" BsonBinaryData values with a GuidRepresentation.

This tagging will be removed in V3.